pkg/config/config.go

```diff
@@ -157,3 +176,16 @@ func WakaHomeDir() (string, error) {
 
 	return "", allerrs
 }
+
+// mutexClock is used to implement mutex.Clock interface.
+type mutexClock struct {
+	delay time.Duration
+}
+
+func (mc *mutexClock) After(time.Duration) <-chan time.Time {
```

Comment:
Is this used?

Reply:
Yep it's needed because `mutex.Spec` expects a `Clock`interface implementaion.